### DISCUSSION TITLE
Design Assets

### DISCUSSION THREAD

**Discussion by TatisLois** [payloadcms/payload#1471] [Feature Requests & Ideas]
## Request

Offer design or branding assets to payload cms for designers and product to use while building out their experience.

## Scenario

I've been working on www.mejor.coach and have been using payload as the foundation of the application. We've been struggling with communicating to designers/non-technical people with what design elements we can use to compose our experience. 

We know in practice we can 1. override and use our own components or 2. override the default styles but for the sake of time we would like to avoid 1 when possible and it's not always clear how/to what extent 2 is applicable. 

It would be great to have design assets that can be used in a tool like figma, where a designer can either A. compose views based on the available default UI, know what areas can be overriden (fonts, colors, layout etc..).

**Comment by jacobsfletch** [comment]
Hey @TatisLois this is great! We are absolutely going to make this happen, and it's actually already in the works. We use Figma internally, and will have a publicly available Payload Master UI for you to use freely.

**Reply by iakshay** [reply]
This would be great, somewhat related request would be to expose the built-in components like table, buttons, toast, text, radio and select fields so those can be used to build custom components. This is possible today by taking dependency on `dist` but would be good to have a nice page (maybe storyboard) documenting usage. Also, making sure future won't break the UI.

**Reply by bombnp** [reply]
I agree with @iakshay. In my case, we're building an internal tool for use within my organization and we need to implement some custom routes to handle custom logic. We're importing from `payload/dist/` but it'd be better if those components are exposed officially.

**Reply by jacobsfletch** [reply]
Related: https://github.com/payloadcms/payload/discussions/1729

**Reply by AndKenneth** [reply]
Sorry for the megabump, but has this been made available @jacobsfletch ? Our designer was about to start creating his own and it would be great if a figma library was available instead.

**Comment by AmyLuckette** [comment]
What's the targeted launch date?

**Comment by Benjapieres** [comment]
Where we can get Payload Master UI?  Thanks
